src/main/java/com/opinta/util/AlphabetUtil.java

```diff
@@ -0,0 +1,52 @@
+package com.opinta.util;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Random;
+
+import com.opinta.entity.ClientType;
+import com.opinta.exception.IncorrectInputDataException;
+
+import static java.lang.String.format;
+import static java.lang.String.valueOf;
+import static java.util.Arrays.asList;
+import static java.util.Locale.ENGLISH;
+
+public class AlphabetUtil {
```

Comment:
Plz, think about the name of the class. It should give idea of what is it doing without dive into it.